src/cbadc/utilities.py

```diff
@@ -232,6 +232,54 @@ def read_byte_stream_from_url(
             pass
     raise StopIteration
 
+def csv_2_control_signal(filename: str, M: int, msb2lsb: bool = False):
+    """Creates an iterator that reads from a CSV file.  
+
+    Parameters
+    ----------
+    filename : `str`
+        filename for input file
+    M : `int`
+        number of controls
+    msb2lsb : `bool`
+        set order of control bits. Default is LSB to MSB
```

Comment:
let's write out the abbreviations.

least significant bit (LSB)

Reply:
good point